doc/sphinx_source/tutorials/tlssetup.rst

```diff
@@ -59,9 +59,9 @@ Connecting to an Eggdrop listening with TLS
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 To connect to a bot listening with TLS, prefix the port number with a '+'. For example::
 
-  .+bot HubBot 1.2.3.4 +5555
```

Comment:
Please revert this file to its original state, thanks

Reply:
done.